### PR TITLE
fix the watch flag

### PR DESCRIPTION
**Pull request by jianglingxia**
maybe better use the format

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/website/6041)

### CODE REVIEW

**Comment by tengqm** [comment]
don't get the benefit of this change...
I'd prefer the current command format.

File: docs/tasks/configure-pod-container/configure-projected-volume-storage.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ Here is the configuration file for the Pod:
 1. Verify that the Pod's Container is running, and then watch for changes to
 the Pod:
 
-       kubectl get --watch pod test-projected-volume
+       kubectl get pod test-projected-volume -w
```
Comment (heckj) [inline docs/tasks/configure-pod-container/configure-projected-volume-storage.md]: from a documentation perspective, this is far less understandable to me. So while correct, I don't think the terse-ness that the short option provides is valuable in this case

Comment (chenopis) [reply docs/tasks/configure-pod-container/configure-projected-volume-storage.md]: Agreed. `--watch` is much more readable than `-w` tacked on to the end. I'm going to close this PR.

**Comment by chenopis** [comment]
I don't believe this small change improves the doc, so I'm closing this PR.

**Pull request closed without merging**